Assignments/Program.cs

```diff
@@ -46,30 +46,23 @@ public class T18 {
 
    /// <summary>Returns the Nth Armstrong number of given input</summary>
    public static int NthArmstrongNum (int input) {
-      int nCount = 0;
-      for (int i = 0; nCount <= input; i++) {
-         if (IsArmsNum (i).Item1) {
-            if (!sArmstrong.Contains (i)) sArmstrong.Add (i);
-            nCount++;
-         }
-         if (nCount == input) return i + 1;
+      for (int i = 0; ; i++) {

```

Comment:
Set _task = new () { 10, 9 }_ and check if this works properly.
You are not checking for sArmstrongs.Contains being true.

Reply:
Yes correct.  Setting task = new () { 10, 9 } makes the for loop to run infinitely.
And, the reason for not checking is that we have no use of the bool sArmstrong.Contains (i).